.changeset/curly-seals-give.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
```

Comment:
Code looks good to me, demos should be updated as part of #594 so that they get the correct accordion styles for the view code on dark theme 👍. Leaving this comment here for design approval, feel free to resolve